MovieCatalog/src/main/java/application/models/actors/ActorDTO.java

```diff
@@ -0,0 +1,78 @@
+package application.models.actors;
+
+import application.entities.Movie;
+import application.models.movies.MovieDTO;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class ActorDTO {
+    private String name;
+    private Integer age;
+    private String bio;
+    private List<Movie> movies;
+
+    public ActorDTO() {
+        this.movies = new ArrayList<>();
+    }
+
+    public ActorDTO(String name, Integer age, List<Movie> movies, String bio) {
+        this.name = name;
+        this.age = age;
+        this.movies = movies;
+        this.bio = bio;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public Integer getAge() {
+        return age;
+    }
+
+    public void setAge(Integer age) {
+        this.age = age;
+    }
+
+    public List<Movie> getMovies() {
+        return movies;
+    }
+
+    public void setMovies(List<Movie> movies) {
+        this.movies = movies;
+    }
+
+    public String getBio() {
+        return bio;
+    }
+
+    public void setBio(String bio) {
+        this.bio = bio;
+    }
+
```

Comment:
Should also implement hashCode method. When  implementing equals , hashCode should always be implemented!